nasa-gibs/worldview
# WV-2943: Node/Lockfile Version update

## Description

Updating Node to version 20.9.0 and the package-lock.json files `lockfileVersion` to 3. 

## How To Test

1. Take the time to update your local environment to `Node` version 20.9.0 and install `npm` at version 10.0.1. `npm install -g npm@10.1.0`
2. `git checkout wv-2943`
3. `npm ci`
4. `npm run watch`
5. Verify that WV runs properly

@nasa-gibs/worldview
